CHANGELOG.md

```diff
@@ -37,6 +37,10 @@ to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
   
 * When reconfiguring the author, warn that the working copy won't be updated
 
+* `jj fix` now allows fixing unchanged files with the `--all-files` flag. This
+  can be used to more easily introduce automatic formatting changes in a new
+  commit separate from other changes.
+
```

Comment:
Don't forget to move this to the new "Unreleased" section after syncing this commit with main